server/Dockerfile

```diff
@@ -6,13 +6,14 @@ VOLUME /timely-server-src
 VOLUME /timely
 
 RUN yum -y update && \
-  yum -y install java-1.8.0-openjdk-devel maven openssl apr && \
-  mkdir -p /timely/bin
+  yum -y install java-1.8.0-openjdk-devel maven openssl apr wget && \
+  mkdir -p /timely/bin && \
+  wget -O /usr/lib/jvm/jre/lib/ext/bcprov-jdk15on-155.jar https://www.bouncycastle.org/download/bcprov-jdk15on-155.jar && \
+  echo "security.provider.9=org.bouncycastle.jce.provider.BouncyCastleProvider" >> /usr/lib/jvm/jre/lib/security/java.security
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.91-1.b14.el7_2.x86_64
+ENV JAVA_HOME /usr/lib/jvm/java
```

Comment:
If Java 8 is specifically desired, it'd be better to use: /usr/lib/jvm/java-1.8.0-openjdk

I'm also partial to using `$(dirname $(dirname $(readlink -e $(which java))))` (or `javac`, if JDK is required for JAVA_HOME)